bddl/activity_definitions/make-warm-drinks-for-the-holidays/problem0.bddl

```diff
@@ -6,7 +6,7 @@
         honey.n.01_1 - honey.n.01
         lemon_juice.n.01_1 - lemon_juice.n.01
         water.n.06_1 - water.n.06
-        beverage.n.01_1 - beverage.n.01
+        hot_toddy.n.01_1 - hot_toddy.n.01
```

Comment:
you might need to update the csv for this task later, just a heads up

Reply:
did it on friday!